### ISSUE TITLE
Docker-compose up error

### ISSUE THREAD

**Issue by stronquens** [heartexlabs/label-studio#4971]
**Describe the bug**
After cloning the repo in branch develop, docker-compose do not work. The persistent storage starts perfect but nginx and app keeps restarting all the time because do not finds something.

**To Reproduce**
1. Download repo
2. Docker-compose up or docker compose -f docker-compose.yml -f docker-compose.minio.yml up

**Expected behavior**
All containers up and running

**Ouput logs**
label-studio-app-1         | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/app/11-configure-custom-cabundle.sh
label-studio-app-1         | /label-studio/deploy/docker-entrypoint.d/app/11-configure-custom-cabundle.sh: line 1: ../common/11-configure-custom-cabundle.sh: No such file or directory
label-studio-app-1 exited with code 127
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Looking for init scripts in /label-studio/deploy/docker-entrypoint.d/nginx/
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh
label-studio-nginx-1       | /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh: line 1: ../common/10-configure-nginx.sh: No such file or directory
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Looking for init scripts in /label-studio/deploy/docker-entrypoint.d/nginx/
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh
label-studio-nginx-1       | /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh: line 1: ../common/10-configure-nginx.sh: No such file or directory
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Looking for init scripts in /label-studio/deploy/docker-entrypoint.d/nginx/
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh
label-studio-nginx-1       | /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh: line 1: ../common/10-configure-nginx.sh: No such file or directory
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Looking for init scripts in /label-studio/deploy/docker-entrypoint.d/nginx/
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh
label-studio-nginx-1       | /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh: line 1: ../common/10-configure-nginx.sh: No such file or directory
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Looking for init scripts in /label-studio/deploy/docker-entrypoint.d/nginx/
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh
label-studio-nginx-1       | /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh: line 1: ../common/10-configure-nginx.sh: No such file or directory
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Looking for init scripts in /label-studio/deploy/docker-entrypoint.d/nginx/
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh
label-studio-nginx-1       | /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh: line 1: ../common/10-configure-nginx.sh: No such file or directory
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Looking for init scripts in /label-studio/deploy/docker-entrypoint.d/nginx/
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh
label-studio-nginx-1       | /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh: line 1: ../common/10-configure-nginx.sh: No such file or directory
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Looking for init scripts in /label-studio/deploy/docker-entrypoint.d/nginx/
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh
label-studio-nginx-1       | /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh: line 1: ../common/10-configure-nginx.sh: No such file or directory
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Looking for init scripts in /label-studio/deploy/docker-entrypoint.d/nginx/
label-studio-nginx-1       | ./deploy/docker-entrypoint.sh: Launching /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh
label-studio-nginx-1       | /label-studio/deploy/docker-entrypoint.d/nginx/10-configure-nginx.sh: line 1: ../common/10-configure-nginx.sh: No such file or directory

**Comment by hogepodge** [comment]
We'll look into reproducing this and will see what we can come up with. Thanks.

**Comment by hogepodge** [comment]
@stronquens, we're not able to reproduce this locally, and think that there might be an issue with your configuration.

**Comment by stronquens** [comment]
I will check it.
I use Windows 11 and Docker descktop v4.11.1

**Comment by vitwei** [comment]
same problem 
it hard to reproducing
maybe is a image problem 
i try to rebuild my docker image and some time this problem will occur

**Comment by iam-dev0** [comment]
I am encountering the same problem today, but I updated docker-compose. Can you guys spot any issues in this compose file?
Note: If I run simple contain from DockerFile it does work.

```
version: "3.9"
services:
  nginx:
    build: .
    image: label_studio:0.0.1  # Using your local image
    restart: unless-stopped
    ports:
      - "8080:8085"
      - "8081:8086"
    depends_on:
      - app
    environment:
      - LABEL_STUDIO_HOST=${LABEL_STUDIO_HOST:-}
    #   Optional: Specify SSL termination certificate & key
    #   Just drop your cert.pem and cert.key into folder 'deploy/nginx/certs'
    #      - NGINX_SSL_CERT=/certs/cert.pem
    #      - NGINX_SSL_CERT_KEY=/certs/cert.key
    volumes:
      - ./mydata:/label-studio/data:rw
      - ./deploy/nginx/certs:/certs:ro
    #   Optional: Override nginx default conf
    #      - ./deploy/my.conf:/etc/nginx/nginx.conf
    command: nginx

  app:
    stdin_open: true
    tty: true
    build: .
    image: label_studio:0.0.1  # Using your local image
    restart: unless-stopped
    expose:
      - "8000"
    environment:
      - DJANGO_DB=default
      - POSTGRE_NAME=label_studio  # Replace with your local DB name
      - POSTGRE_USER=postgres      # Replace with your local DB user
      - POSTGRE_PASSWORD=password  # Replace with your local DB password
      - POSTGRE_PORT=5432                     # Default PostgreSQL port
      - POSTGRE_HOST=host.docker.internal     # Allows Docker containers to access services on the host
      - LABEL_STUDIO_HOST=${LABEL_STUDIO_HOST:-}
      - JSON_LOG=1
    #      - LOG_LEVEL=DEBUG
    volumes:
      - ./mydata:/label-studio/data:rw
    command: label-studio-uwsgi

```

**Comment by iam-dev0** [comment]
After extensive troubleshooting, I have identified the cause of our recent issues with the Docker setup. The problem stemmed from the use of symbolic links within the docker-entrypoint.d directory. It appears that symbolic links do not behave consistently when creating Docker images on Windows, due to differences in how file links are handled between Windows and Unix-based systems.

As a test, I replaced all symbolic links with their actual files. This modification successfully resolved the issue, confirming that the symbolic links were the root cause.

Happy coding!
